server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +211,50 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature, $appointmentId) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+	
+	if($virtualConsent === 'true' && $virtualConsent !== null && $signature !== null && $signature !== '' && $appointmentId !== null) {
+		$reviewConsent = $reviewConsent === 'true' ? 1 : 0;
+		$virtualConsent = $virtualConsent === 'true' ? 1 : 0;
+		try {
+			$DB_CONN->beginTransaction();
+
+			insertConsent($reviewConsent, $virtualConsent, $signature, $appointmentId);
+			$DB_CONN->commit();
+
+			$response['success'] = true;
```

Comment:
I think you forgot to put the consented response in here? It looks like you expect a `'consented'` in there according to uploadDocumentsController.js line 60.

Reply:
Yeah, that's right.